data/nlu/nlu.md

```diff
@@ -3765,6 +3765,14 @@
 - where to start?
 - yes i wanna know more about rasa [nlu](product)
 - I am trying to build a bot using rasa
```

Comment:
we should probably annotate rasa as a product as well then everywhere.

Reply:
It seems like `rasa` is used a lot though in contexts where it isn't about the oss product - maybe better to turn `rasa open source` into its own entity? Since we started referring to it that way more recently

Reply:
hm ok. but then i would say leaving the entity annotated as it is now is fine, will approve the PR

Reply:
maybe if we notice a change in how users use it in conversations later we can change it